### PR TITLE
Generate full pdbs for desktop packages

### LINKED ISSUE #1548: Ensure we have full pdbs for binaries that we ship in VS
The SDK defaults to producing portable pdbs. VS currently requires regular pdbs for symbol archival. We'll need to ensure we produce the right kind of pdbs for these for the 15.5 Razor release.

### PR DESCRIPTION
**Pull request by pranavkm**
Fixes #1548

### CODE REVIEW

File: build/common.props [hunk 1]
```diff
@@ -12,6 +12,9 @@
     <PublicSign Condition="'$(OS)' != 'Windows_NT'">true</PublicSign>
     <VersionSuffix Condition="'$(VersionSuffix)'!='' AND '$(BuildNumber)' != ''">$(VersionSuffix)-$(BuildNumber)</VersionSuffix>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+
+    <!-- Generate full pdbs for desktop targeting projects on platforms that support generating full pdbs. -->
+    <DebugType Condition="'$(OS)'=='Windows_NT' AND '$(TargetFramework)'=='net46'">full</DebugType>
```
Comment (rynowak) [inline build/common.props]: Is it possible to make it so that this will make noise if we target another `net4X`?

**Comment by rynowak** [comment]
We need to port this to rel/15.3 also

File: build/common.props (hunk 1 above)
Comment (pranavkm) [inline build/common.props]: Oddly using `"TargetFrameworkIdentifier"==".NETFramework"` didn't generate full pdbs. `TargetFramework` was the only thing that worked. We have other instances of `net46` in our proj files so it's unlikely we'll miss this when we change update.

**Comment by pranavkm** [comment]
Ported to 15.3 here: https://github.com/aspnet/Razor/commit/e30b09f08e554e0b860d8e1fdf4dd9e08e6f5293

**Pull request merged**